### PR TITLE
Allowing to set a structure label via loading from url api

### PR DESCRIPTION
**Pull request by valasatava**
Needed to fully resolve issues that might occur in scenarios similar to https://rcsbpdb.atlassian.net/browse/RO-3915

Also, a rcsb-molstar release is needed for pecos-app

### CODE REVIEW

File: src/viewer/index.html [hunk 1]
```diff
@@ -653,6 +653,11 @@ <h2> RCSB PDB Mol* Viewer - Test Page</h2>
                             assemblyId: '1'
                         }
                     }
+                },
+                {
+                    id: '',
```
Comment (JonStargaryen) [inline src/viewer/index.html:658]: Artifact?

File: src/viewer/index.ts [hunk 2]
```diff
@@ -297,7 +297,7 @@ export class Viewer {
         return out;
     }
 
-    loadStructureFromUrl<P, S>(url: string, format: BuiltInTrajectoryFormat, isBinary: boolean, config?: {props?: PresetProps; matrix?: Mat4; reprProvider?: TrajectoryHierarchyPresetProvider<P, S>, params?: P}) {
+    loadStructureFromUrl<P, S>(url: string, format: BuiltInTrajectoryFormat, isBinary: boolean, config?: {props?: PresetProps & { structureLabel?: string }; matrix?: Mat4; reprProvider?: TrajectoryHierarchyPresetProvider<P, S>, params?: P}) {
```
Comment (JonStargaryen) [inline src/viewer/index.ts:300]: There's `dataLabel` available for `loadStructureFromData`, could consider naming this the same, but either way seems fine to me.

**Review by JonStargaryen — CHANGES_REQUESTED** [review]
Thanks, LGTM overall. index.html contains some undesired changes.

File: src/viewer/index.html (hunk 1 above)
Comment (valasatava) [reply src/viewer/index.html:658]: Yes, totally. Thanks!

File: src/viewer/index.ts (hunk 2 above)
Comment (valasatava) [reply src/viewer/index.ts:300]: Does `dataLabel` serve similar purpose?

Comment (JonStargaryen) [reply src/viewer/index.ts:300]: Both get passed to the builder in the same fashion: https://github.com/molstar/rcsb-molstar/blob/db1347de0f549634a94d962fee346f82561934da/src/viewer/helpers/model.ts#L29

Though I don't think we use that functionality.

**Review by JonStargaryen — APPROVED** [review]
Thanks for the cleanup & changes.

**Pull request merged**